dist/function.py

```diff
@@ -189,8 +189,161 @@ def function_declaration():
             tfile.close()
             compare_srcfile(tmp_file, name)
 
+# A function definition.
+class FunctionDefinition:
+    def __init__(self, name, module, source):
+        self.name = name
+        self.module = module
+        self.source = source
+        self.used_outside_of_file = False
+        self.used_outside_of_module = False
```

Comment:
A possible performance improvement.
I wonder if there should be two more fields: self.global_visible for __wt names and self.module_visible for __wti names. Then check these instead of using startswith(). Which is faster?

Reply:
Fixed in [acfc825](https://github.com/wiredtiger/wiredtiger/pull/10667/commits/acfc825f968f01aa6feb302fe1f275bfe33ad7f3).